src/terminal/color-scheme.c

```diff
@@ -70,7 +70,7 @@ static int guac_terminal_color_scheme_compare_token(const char* str_start,
  *     Address of a pointer to the end of the string, after the last character.
  *     On return, the pointer is moved back to before any trailing spaces.
  */
-static void guac_terminal_color_scheme_strip_spaces(const char** str_start,
+static void guac_terminal_get_color_scheme_strip_spaces(const char** str_start,
```

Comment:
Similar here: this isn't a getter, but a function which operates directly on the given values. It's in the "color_scheme" namespace beneath the "guac_terminal" namespace, thus has a `guac_terminal_color_scheme_` prefix, and has the descriptive name `strip_spaces` after that describing the action performed. I think that original naming was correct.

Reply:
Yeah, this shouldn't have been renamed. Putting it back.